cylc/flow/cycling/iso8601.py

```diff
@@ -737,7 +737,15 @@ def prev_next(
     }
 
     for my_time in str_points:
-        parsed_point = parser.parse(my_time.strip())
+        try:
+            parsed_point = parser.parse(my_time.strip())
+        except ValueError:
+            suggest = my_time.replace(',', ';')
+            raise CylcConfigError(
+                f'Invalid offset: {my_time}:'
+                f' Offset lists are semicolon separated, try {suggest}'
+            )
```

Comment:
This is a bit eager, e.g. it will match `next(horse)`. This is because `ISO8601SyntaxError` is a subclass of `ValueError`. 
I would suggest re-raising the existing exception if it is an instance of `ISO8601SyntaxError`